rustup-init.sh

```diff
@@ -33,7 +33,7 @@ RUSTUP_QUIET=no
 # NOTICE: If you change anything here, please make the same changes in setup_mode.rs
 usage() {
     cat <<EOF
-rustup-init 1.28.1 (83ac5d9ec 2025-03-04)
```

Comment:
You don't need to update this file: it is part of our release process.

Reply:
I reverted that